.github/workflows/nim_test.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: nim_test
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
```

Comment:
Not sure we want workflow dispatch when it already runs automatically. 🤔

Reply:
_Just in case_. Having this extra line does not hurt and it helps to avoid some _blank_ PR's just to run some workflow (cf. TheAlgorithms/Go#638)